src/app/components/ListTitle/ListTitle.jsx

```diff
@@ -6,26 +6,24 @@ const ListTitle = (props) => {
   const booksfound = `${props.picksCount} Book${props.picksCount === 1 ? '' : 's'} Found`;
   const idPrefix = props.idPrefix ? `${props.idPrefix}-` : '';
 
+  // General check to see if the year is available.
   if (!displayDate.year) {
     return null;
   }
 
-  if (props.displayType !== 'staff-picks') {
-    return (
-      <h2 id={`${idPrefix}list-title`} tabIndex="0">
-        {displayDate.year} Picks
-        <span className="pick-count">{booksfound}</span>
-      </h2>
-    );
-  }
+  const isStaffPicks = (props.displayType !== 'staff-picks');
```

Comment:
The name and what it represents is opposite. If `props.displayType` is `ya`, `isStaffPicks` will be true.

Reply:
Good catch.